builtin/logical/pki/ca_test.go

```diff
@@ -689,9 +689,10 @@ func runSteps(t *testing.T, rootB, intB *backend, client *api.Client, rootName,
 		// Run with a short safety buffer and both set to true, both should be cleared
 		{
 			resp, err := client.Logical().Write(rootName+"tidy", map[string]interface{}{
-				"safety_buffer":      "1s",
-				"tidy_cert_store":    true,
-				"tidy_revoked_certs": true,
+				"safety_buffer":         "1s",
+				"revoked_safety_buffer": "1s",
```

Comment:
This is a bit weird... There is a nil deference error when `revoked_safety_buffer` is not defined although this should not be the case since this is already handled in the function `pathTidyWrite`. 

I'm a lil confused, I'm not sure how to read the values from this endpoint. When I read the parameters by `client.Logical().Read(rootName + "config/auto-tidy")`, the values are the default values set (i.e., 72h) which makes sense since I haven't written the buffer values to this endpoint.

Reply:
Yes I think this test was failing because I was defaulting revoked safety buffer to safety buffers default value instead of safety buffer itself when safety buffer was not specified... the test passes now :D